test/fixtures/globala.js

```diff
@@ -0,0 +1,5 @@
+global.a = 'test';
+process.stdin.on('data', function(chunk) {
```

Comment:
I just ran this locally without this block of code and it still passed. Are you sure this is needed?